source/meta.tex

```diff
@@ -5618,7 +5618,7 @@
 appear in the order in which they are declared.
 \begin{note}
 Base classes are not members.
-Implicitly-declared special members
+Implicitly-declared special members and comparison operator member functions
```

Comment:
Are there any "special members" that are not "member functions"?  Why the difference in presentation vs. "comparison operator member functions"?

Maybe just "Implicitly-declared member functions appear ..." ?

(I don't think we normatively say whether implicit `operator<=>` is declared before or after the special members, so there should be no hint in the note about their relative order.)